cmd/crawler/config/config.go

```diff
@@ -0,0 +1,196 @@
+package config
```

Comment:
It's better to merge with https://github.com/bCoder778/qitmeer/blob/0.10-dev/cmd/crawler/node/config.go

Reply:
ok